Makefile

```diff
@@ -61,6 +63,7 @@ FOGLAMP_SCRIPT_SRC         := scripts/foglamp
 
 # SCRIPTS TO INSTALL IN SCRIPTS DIR
 COMMON_SCRIPTS_SRC         := scripts/common
+CORE_SCRIPTS_SRC           := scripts/core/selfssl
```

Comment:
Please rename this to SELFSSL_SCRIPT_SRC as it refers to a very particular source file.